cluster/src/dunit/scala/io/snappydata/cluster/ClusterManagerTestBase.scala

```diff
@@ -121,7 +121,8 @@ abstract class ClusterManagerTestBase(s: String)
       }
     }
 
-    Array(vm0, vm1, vm2).foreach(_.invoke(startNode))
+    vm0.invoke(startNode)
+    Array(vm1, vm2).map(_.invokeAsync(startNode)).foreach(_.getResult)
```

Comment:
Small things but great improvement to save time. Just curious, overall how much time it is saving in dunits approx. 10 to 20 minutes or less?